contrib/whois/template.py

```diff
@@ -0,0 +1,47 @@
+pkgname = "whois"
+pkgver = "5.5.18"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_build_args = [
+    "CONFIG_FILE=/etc/whois.conf",
+    "HAVE_ICONV=1",
+]
+make_install_target = "install-whois"
+make_install_args = ["install-mkpasswd"]
+hostmakedepends = [
+    "gettext",
+    "gmake",
+    "perl",
+    "pkgconf",
+]
+makedepends = [
+    "libidn2-devel",
+]
+pkgdesc = "Intelligent WHOIS Client"
+maintainer = "psykose <alice@ayaya.dev>"
+license = "GPL-2.0-or-later"
+url = "https://github.com/rfc1036/whois"
+source = f"https://github.com/rfc1036/whois/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "f0ecc280b5c7130dd8fe4bd7be6acefe32481a2c29aacb1f5262800b6c79a01b"
+hardening = ["vis", "cfi"]
+# no tests
+options = ["!check"]
+
+tool_flags = {
+    "CFLAGS": ["-DHAVE_GETOPT_LONG", "-DHAVE_GETADDRINFO", "-DHAVE_SHA_CRYPT"]
+}
+
+
+def init_install(self):
+    self.make_install_args.append(f"INSTALL_ROOT={self.chroot_destdir}")
+
+
+@subpackage("mkpasswd")
+def _mkpasswd(self):
+    # conflicting /usr/bin/mkpasswd
```

Comment:
the manpage also conflicts. not sure what to do about that

the actual mkpasswd (and manpage) are completely different, just happen to have the same name..

Reply:
i'll probably change expect to do the same as debian does, i.e. call the scripts expect_foo

you can drop the subpackage